shared_model/interfaces/common_objects/signable_hash.hpp

```diff
@@ -54,8 +54,6 @@ namespace shared_model {
      * limitations: it requires to have write access for elements for some
```

Comment:
please remove comments as well

Reply:
Also please rework `size_t operator()(const types::SignatureType &sig) const` so that it gets `const Signature &` without PolymorphicWrapper. I think there is implicit PolymorphicWrapper constructor call in hashing.